src/main/java/model/input/file/PlainTextFile.java

```diff
@@ -43,15 +45,14 @@ public DataTable createDataTable() throws IOException {
 			if (hasFirstRowAsHeader()) {
 				String headers = scanner.nextLine();
 				String[] sections = headers.split(delimiter);
-				for (int i = 0; i < getColumnTypes().length; i++) {
-					getColumns().put(sections[i], getColumnTypes()[i]);
+				for (int i = 0; i < getColumns().size(); i++) {
```

Comment:
het is een vrij lange methode zonder line breaks wat hem erg slecht leesbaar maakt. Dus je wilt deze methode inkorten. Ik zou bijvoorbeeld de body van de if in een aparte methode gooien